package.json

```diff
@@ -2,9 +2,8 @@
   "scripts": {
     "layer0": "layer0",
     "layer0:dev": "layer0 dev",
-    "layer0:build": "npm run craco:build && layer0 build",
-    "layer0:deploy": "npm run craco:build && layer0 deploy layer0-docs",
-    "deploy": "npm run craco:build && layer0 deploy",
+    "layer0:build": "layer0 build",
+    "deploy": "layer0 deploy",
```

Comment:
let's keep this as `layer0:deploy`, gh action updated as necessary too 🙏

Reply:
sounds good, making the changes!